Add reply arrow example (#3546)
mattermost/docs: source/help/messaging/sending-messages.rst

Replying to Messages
---------------------

> Click the reply link (arrow icon |replyarrow| that appears when hovering over a message) next to any message. This keeps conversations organized in threads. Replies are indented slightly in the center pane to indicate that they are child messages of a parent message.

Reviewer: ```suggestion
Hover over any message and click the reply link (arrow icon |replyarrow|). This organizes the conversation into a thread. Replies are indented slightly in the center pane to indicate that they are child messages of a parent message.
```
